src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands/NewAzureRmAnalysisServicesServer.cs

```diff
@@ -49,19 +50,23 @@ public class NewAnalysisServicesServer : AnalysisServicesCmdletBase
                 "Name of the Sku used to create the server"
             )]
         [ValidateNotNullOrEmpty]
-        [ValidateSet("B1", "B2", "S0", "S1", "S2", "S4", "D1")]
         public string Sku { get; set; }
 
         [Parameter(ValueFromPipelineByPropertyName = true, Position = 4, Mandatory = false,
             HelpMessage = "A string,string dictionary of tags associated with this server")]
         [ValidateNotNull]
         public Hashtable Tag { get; set; }
 
-        [Parameter(ValueFromPipelineByPropertyName = true, Position = 4, Mandatory = false,
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 5, Mandatory = false,
```

Comment:
@taiwu this is causing the build to fail as it is considered a breaking change, but it is a false positive since a user would have never been able to supply a value for this position (since it shared the position with `Tag`). You can suppress this exception by copying the line from the `BreakingChangeIssues.csv` file from the build and appending it to the end of [this file](https://github.com/Azure/azure-powershell/blob/preview/tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv).

_Note_: you will need to copy and paste the line from the csv file using a text editor and not Excel since this will cause the quotations to be removed between each item, causing an error to be thrown.